test/integration/scheduler/scheduler_test.go

```diff
@@ -82,181 +79,6 @@ func PriorityTwo(pod *v1.Pod, nodeNameToInfo map[string]*schedulercache.NodeInfo
 	return []schedulerapi.HostPriority{}, nil
 }
 
-// TestSchedulerCreationFromConfigMap verifies that scheduler can be created
```

Comment:
Why is this test deleted? We need a test on creating scheduler from a ConfigMap.

Reply:
Creating a scheduler from a serialized policy is already covered in the [TestCreateFromConfig  unit test](https://github.com/ironcladlou/kubernetes/blob/kube-scheduler-config/plugin/pkg/scheduler/factory/factory_test.go#L76). A new additional unit test to cover [makeSchedulerConfig](https://github.com/ironcladlou/kubernetes/blob/kube-scheduler-config/plugin/cmd/kube-scheduler/app/server.go#L647) would also be useful to ensure we actually attempt to follow the code paths in TestCreateFromConfig when set up to use the config map source.

The integration test relies on assertions against internal state of the scheduler exposed for the sole purpose of this particular integration test.

Switching to unit tests in this case could provide comparable coverage but faster and without exposing state unnecessarily.

Happy to try and restore this if you're unconvinced, please let me know what you think.

Reply:
The integration test was also testing the fact that scheduler reads the config from etcd and parses it. This functionality is not tested in the unit test.

Reply:
The two configmap integration tests have been restored.